### ISSUE TITLE
Internal Server Error when exporting a dataset

### ISSUE THREAD

**Issue by japel** [openvinotoolkit/cvat#4380]
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

### Expected Behaviour

Generated dataset archive should be sent to the client for download.
### Current Behaviour

On chrome a "dataset.html" is attemped to be downloaded, but fails with "Failed - Server problem".

### Possible Solution

Python 3 renamed the unicode type to str, the old str type has been replaced by bytes. Seems like the uderlying lib is still using unicode which fails in python3 installations. Perhaps a newer version of the lib should be used? django-sendfile2 might be an option, but have not tested it yet.

### Steps to Reproduce (for bugs)

1. Create a project
2. Create a Task
3. Export task dataset (including images)

### Context

Just exporting a previously created dataset.
### Your Environment

- Git hash commit (`git log -1`): 967b0fee2c1c5028ad5f8e888b4eb5c138fe2b1e
- Docker version `docker version` (e.g. Docker 17.0.05): 
- Are you using Docker Swarm or Kubernetes? Kubernetes
- Operating System and version (e.g. Linux, Windows, MacOS): Linux
- Code example or link to GitHub repo or gist to reproduce problem:
- Other diagnostic information / logs:
Logs from `cvat_server` container
```
ERROR - 2022-02-23 12:17:13,334 - log - Internal Server Error: /api/v1/tasks/2/dataset
 Traceback (most recent call last):
   File "/opt/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
     response = get_response(request)
   File "/opt/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
     response = wrapped_callback(request, *callback_args, **callback_kwargs)
   File "/opt/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
     return view_func(*args, **kwargs)
   File "/opt/venv/lib/python3.8/site-packages/rest_framework/viewsets.py", line 114, in view
     return self.dispatch(request, *args, **kwargs)
   File "/opt/venv/lib/python3.8/site-packages/rest_framework/views.py", line 505, in dispatch
     response = self.handle_exception(exc)
   File "/opt/venv/lib/python3.8/site-packages/rest_framework/views.py", line 465, in handle_exception
     self.raise_uncaught_exception(exc)
   File "/opt/venv/lib/python3.8/site-packages/rest_framework/views.py", line 476, in raise_uncaught_exception
     raise exc
   File "/opt/venv/lib/python3.8/site-packages/rest_framework/views.py", line 502, in dispatch
     response = handler(request, *args, **kwargs)
   File "/home/django/cvat/apps/engine/views.py", line 902, in dataset_export
     return _export_annotations(db_instance=db_task,
   File "/home/django/cvat/apps/engine/views.py", line 1616, in _export_annotations
     return sendfile(request, file_path, attachment=True,
   File "/opt/venv/lib/python3.8/site-packages/sendfile/__init__.py", line 68, in sendfile
     response = _sendfile(request, filename, mimetype=mimetype)
   File "/opt/venv/lib/python3.8/site-packages/sendfile/backends/xsendfile.py", line 6, in sendfile
     response['X-Sendfile'] = unicode(filename).encode('utf-8')
 NameError: name 'unicode' is not defined
```

**Comment by japel** [comment]
Seems like this is going to be solved in 2.0.0

**Comment by azhavoro** [comment]
@japel correct, fixed in the develop
